### DISCUSSION TITLE
Is all the features of Chatwoot are avialable if we self host it ?

### DISCUSSION THREAD

**Discussion by hasnat-afzal-pk** [chatwoot/chatwoot#9630] [Self hosted] [answered]
If I self host the chatwoot on my own clusters Do I have access to all enterprise features?

**Comment by Daniel15** [comment, marked as answer]
You'd need to pay for enterprise to get the enterprise features. Pricing is listed here: https://www.chatwoot.com/pricing/self-hosted-plans
